helpdesk_mgmt/templates/helpdesk_ticket_templates.xml

```diff
@@ -56,71 +50,86 @@
             <t t-call="portal.portal_searchbar">
                 <t t-set="title">Tickets</t>
             </t>
-            <form method="POST" t-attf-action="/new/ticket">
-                <button
-                    name="create_new_ticket"
-                    type="action"
-                    class="btn btn-primary"
-                    style="float: right; margin-right: 0px; margin-bottom: 5px;"
-                >New Ticket</button>
-                <input
-                    type="hidden"
-                    name="csrf_token"
-                    t-att-value="request.csrf_token()"
-                />
-            </form>
             <t t-if="not grouped_tickets">
                 <div class="alert alert-warning mt8" role="alert">
                     <p>No tickets found.</p>
                 </div>
             </t>
+            <div class="d-flex pt-1">
```

Comment:
This change should go and debated in a separate commit.

Reply:
Thanks for the review  @pedrobaeza 
Considering your request and the fact that I squashed everything ;-) I think I will rewrite all this PR to fullfill your requirements .
Do you have additional advice you want me to isolate in a commit ?
Regards

Reply:
Yeah, when I said "squash the commit history a bit" was for avoiding noise, but grouping things in a logical way for being easily reviewed isolatedly. There's no absolute choice here.

Reply:
And if grouping together, the first line of the commit message should reflect everything the commit contains.